crates/l2/monitor/widget/batches.rs

```diff
@@ -98,7 +85,7 @@ impl BatchesTable {
         let last_l2_batch_number = eth_client
             .get_last_committed_batch(on_chain_proposer_address)
             .await
-            .expect("Failed to get latest L2 batch");
+            .map_err(|_| MonitorError::GetLatestBatch)?;
```

Comment:
Why not use `MonitorError::EthClientError`?
or:
```suggestion
            .map_err(MonitorError::from)?;
```